tola/activity_proxy.py

```diff
@@ -49,9 +49,26 @@ def get_by_url(url):
     headers = _get_headers()
     response = requests.get(url, headers=headers)
     content = {}
-
     if response.status_code == 200:
         content = json.loads(response.content)
     else:
         logger.warn('{}: {}'.format(response.status_code, response.content))
     return content
+
+
+def get_workflowlevel1s(user):
+    try:
+        params = {
+            'workflow_user__tola_user_uuid': user.tola_user.tola_user_uuid
+        }
+        wfteams = get_workflowteams(**params)
+        wfl1_uuids = []
+        for wfteam in wfteams:
```

Comment:
- Why did you choose this approach?
- Is it not possible to do in a more efficient way?

Reply:
Workflows and users associated via workflowteams. So, I need to call get_workflowteams to access workflowlevel1s. I applied to same implementation in SiloForm. Actually, workflowteam model has also user_access field (many ), I think that is more useful but I guess we are not using it.

Reply:
- If we have 100k workflowteams, we're gonna create 100k requests. So, making many requests isn't good. Do you agree with me? Don't you think that there's a better way to do it, like only one request?
- I didn't see the field `user_access`. Can you be more precise? https://github.com/toladata/TolaActivity/blob/master/workflow/models.py#L706

Reply:
Sorry, workflowlevel1 model has user_access field https://github.com/toladata/TolaActivity/blob/f6c4e1d61883c754cc3e45af8d409504cd717157/workflow/models.py#L638 . So, we can get users workflowlevel1's at one request. 

But currently we are using workflowteam to access workflowlevel1s and we need 2 request to access workflowlevel1s uuid. You are right it is not good solution.  Another way is creating new endpoint in ActivityAPI for this process. We can create another ticket for this.